tailor_distro/debian_templates/rules.j2

```diff
@@ -71,6 +71,6 @@ override_dh_shlibdeps:
 
 override_dh_installdeb:
 	# Fixup absolute and relative paths for installation target into /opt, don't touch .so libs
-	find . -type f ! -name '*.so*' -exec sed -ri "s|($(CURDIR)/)?debian/tmp/opt|/opt|g" {} ";" && \
+	find . -type f ! -name '*.so*' -exec grep -I -q . {} \; -print0 | xargs -0 sed -ri "s|($(CURDIR)/)?debian/tmp/opt|/opt|g" && \
```

Comment:
you may want to drop `! -name '*.so*'` since it's covered by the binary check

Reply:
Done